framework/src/main/java/org/checkerframework/framework/util/dependenttypes/DependentTypesHelper.java

```diff
@@ -235,12 +257,6 @@ private void viewpointAdaptExecutable(
             return;
         }
 
-        JavaExpression receiver = JavaExpression.getReceiver(tree);
-        List<JavaExpression> argsJe = argumentTreesToJavaExpressions(tree, methodType, argTrees);
-
-        JavaExpressionContext context =
-                new JavaExpressionContext(receiver, argsJe, factory.getChecker());
-
         // methodType cannot be viewpoint adapted directly because it is the type post type variable
         // substitution.  Dependent type annotations on type arguments cannot be
         // viewpoint adapted along with the dependent type annotations that are on the method
```

Comment:
The comment below

> If the type of List.get is viewpoint adapted for the invocation "list.get(0)", then
> methodType would be @KeyFor("map") String get(int).

is written as if it is a counterfactual.  What is its relation to the actual situation?  Please clarify.

Also,

> use the type for the method (viewpointAdaptedType) and viewpoint adapt that
> type.

makes it sound like viewpoint adaptation is being done twice.